src/views/UserSearch/Filters.js

```diff
@@ -1,38 +1,116 @@
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  get,
+  sortBy,
+} from 'lodash';
 
 import {
-  FilterGroups,
+  Accordion,
+  AccordionSet,
+  FilterAccordionHeader,
 } from '@folio/stripes/components';
 
+import {
+  CheckboxFilter,
+  MultiSelectionFilter,
+} from '@folio/stripes/smart-components';
+
+import { statusFilter } from '../../constants';
+
 export default class Filters extends React.Component {
   static propTypes = {
     activeFilters: PropTypes.object,
+    resources: PropTypes.object.isRequired,
     onChangeHandlers: PropTypes.object.isRequired,
-    config: PropTypes.arrayOf(PropTypes.object),
   };
 
   static defaultProps = {
     activeFilters: {},
   }
 
-  render() {
+  getStaticFilterValues = (items) => {
+    return items.map(item => ({ label: item, value: item }));
+  };
+
+  getValuesFromResources = (type, key, sortByKey = false) => {
+    const items = get(this.props.resources, `${type}.records`, [])
+      .map(item => ({ label: item[key], value: item[key] }));
+
+    return sortByKey ? sortBy(items, key) : items;
+  };
+
+  handleFilterChange = (filter) => {
     const {
       activeFilters,
-      onChangeHandlers: { checkbox, clearGroup },
-      config,
+      onChangeHandlers,
     } = this.props;
 
-    const groupFilters = {};
-    activeFilters.string.split(',').forEach(m => { groupFilters[m] = true; });
+    onChangeHandlers.state({
+      ...activeFilters,
+      [filter.name]: filter.values
+    });
+  };
+
+  render() {
+    const {
+      activeFilters: {
+        active = [],
+        pg = [],
+        tags = [],
+      },
+      onChangeHandlers: { clearGroup },
+    } = this.props;
 
     return (
-      <FilterGroups
-        config={config}
-        filters={groupFilters}
-        onChangeFilter={checkbox}
-        onClearFilter={clearGroup}
-      />
+      <AccordionSet>
+        <Accordion
+          displayClearButton
+          id="users-filter-accordion-status"
+          header={FilterAccordionHeader}
+          label={<FormattedMessage id="ui-users.status" />}
+          separator={false}
+          onClearFilter={() => clearGroup('active')}
+        >
+          <CheckboxFilter
+            dataOptions={this.getStaticFilterValues(statusFilter)}
+            name="active"
+            selectedValues={active}
+            onChange={this.handleFilterChange}
+          />
+        </Accordion>
+        <Accordion
+          displayClearButton
+          id="users-filter-accordion-patron-group"
+          header={FilterAccordionHeader}
+          label={<FormattedMessage id="ui-users.information.patronGroup" />}
+          separator={false}
+          onClearFilter={() => clearGroup('pg')}
+        >
+          <CheckboxFilter
+            dataOptions={this.getValuesFromResources('patronGroups', 'group')}
+            name="pg"
+            selectedValues={pg}
+            onChange={this.handleFilterChange}
+          />
```

Comment:
We sort the tags; why not the patron groups? It feels to me like we should choose between assuming all resources will (1) already be sorted or (2) need to be sorted here.

Reply:
I was asked by @a-mbreaux  about it in similar ticket for ui-requests (sort tags) that is why I've added this ability and did not change existing filters.

Reply:
Either we should expect the list to be sorted (as we do for patron groups) or we should expect to sort it ourselves (as we do for tags). It does not matter which option we choose, but let's be consistent.